### PR TITLE
added SAM and MAM classification criteria

### PR DESCRIPTION
**Pull request by Ninicorn**

### CODE REVIEW

File: docs/source/intervention_models/wasting_treatment/index.rst [hunk 1]
```diff
@@ -62,62 +56,234 @@ in 2012. This is the latest National Guideline for the Management of Acute Malnu
...
 
+:download:`2019 guidelines<guidelines_2019.pdf>`
 
-6-60 months
-+++++++++++
+.. note:: 
+  
+  - Based on the guideline, we will need to make some modelling decisions on how we classifiy SAM, MAM, NONE and think about how it may bias our results. 
+
+  - The ORs and ANDs in the classifications are not very clear or consistent in the guideline. It would be useful if we can clarify with a clinician when it should be OR/AND?
```
Comment (aflaxman) [inline docs/source/intervention_models/wasting_treatment/index.rst:158]: Let's ask Rafael Lozano

File: docs/source/intervention_models/wasting_treatment/index.rst [hunk 2]
```diff
@@ -62,62 +56,234 @@ in 2012. This is the latest National Guideline for the Management of Acute Malnu
...
 
-Moderate acute malnutrition
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
-- Children 6-59 months: WFH <-2 z-score
+:underline:`Classify SAM with or without medical complications`
 
-With medical complications
-^^^^^^^^^^^^^^^^^^^^^^^^^^
+* Any grade of bilateral pitting oedema (+, ++ or +++) OR
+* WFL < -3 zscore OR
```
Comment (aflaxman) [inline docs/source/intervention_models/wasting_treatment/index.rst:171]: WFL = "WHZ" in GBD lingo?

File: docs/source/intervention_models/wasting_treatment/index.rst [hunk 3]
```diff
@@ -62,62 +56,234 @@ in 2012. This is the latest National Guideline for the Management of Acute Malnu
...
+  The WHO classification criteria for infants 0-6mo is 
+
+    1) weight-for-length <–3 Z-scores of the WHO Child Growth Standards median, or
+    2) presence of bilateral pitting oedema 
+
+
+:underline:`Classify MAM`
+
+* MUAC of lactatating mother of infant 0-6 months <23.0 cm
+* WFH ≥-3 to <-2 zscores AND
```
Comment (aflaxman) [inline docs/source/intervention_models/wasting_treatment/index.rst:206]: WFH -> WHZ?

File: docs/source/intervention_models/wasting_treatment/index.rst [hunk 4]
```diff
@@ -62,62 +56,234 @@ in 2012. This is the latest National Guideline for the Management of Acute Malnu
...
+* Clinically well and alert
+
+**Treatment** 
+
+Assess the infant’s feeding and counsel the mother or caregiver on appropriate IYCF practices. Emphasize on establishing effective exclusive
+breastfeeding. If feeding problems, follow up in 5 days. If no feeding problem, follow up in 30 days. Admit the mother to TSFP. Refer the mother for Productive Safety Net Programme (PSNP).
+
+:underline:`Classify no acute malnutrition`
+
+* WFH ≥-2 zscores AND
```
Comment (aflaxman) [inline docs/source/intervention_models/wasting_treatment/index.rst:218]: WHZ

File: docs/source/intervention_models/wasting_treatment/index.rst [hunk 5]
```diff
@@ -62,62 +56,234 @@ in 2012. This is the latest National Guideline for the Management of Acute Malnu
...
+
+.. _2.1:
+
+In children 6-59 months
+++++++++++++++++++++++++
+
+:underline:`Classify SAM with medical complications`
+
+* Any grade of bilateral pitting oedema (+, ++, +++) OR 
+* WFH < -3 zscore OR
```
Comment (aflaxman) [inline docs/source/intervention_models/wasting_treatment/index.rst:233]: WHZ

File: docs/source/intervention_models/wasting_treatment/index.rst [hunk 6]
```diff
@@ -62,62 +56,234 @@ in 2012. This is the latest National Guideline for the Management of Acute Malnu
...
+  - Eye signs of vitamin A deficiency
+
+**Treatment**
+
+Admit to SC
+
+:underline:`Classify SAM without medical complications`
+
+* Bilateral pitting oedema + or ++ OR 
+* WFH <-3 z-scores AND
```
Comment (aflaxman) [inline docs/source/intervention_models/wasting_treatment/index.rst:258]: WHZ

File: docs/source/intervention_models/wasting_treatment/index.rst [hunk 7]
```diff
@@ -62,62 +56,234 @@ in 2012. This is the latest National Guideline for the Management of Acute Malnu
...
+* No medical complications
+* Clinically well and alert
+
+**Treatment**
+
+Admit in OTP
+
+:underline:`Classify MAM`
+
+* WFH ≥ -3 to <-2 z scores AND
```
Comment (aflaxman) [inline docs/source/intervention_models/wasting_treatment/index.rst:269]: WHZ

File: docs/source/intervention_models/wasting_treatment/index.rst [hunk 8]
```diff
@@ -62,62 +56,234 @@ in 2012. This is the latest National Guideline for the Management of Acute Malnu
...
+* No medical complications
+* Clinically well and alert
+
+**Treatment**
+
+Admit in TSFP and counsel on appropriate IYCF practices.
+
+:underline:`Classify no acute malnutrition`
+
+* WFH ≥ -2 z score AND
```
Comment (aflaxman) [inline docs/source/intervention_models/wasting_treatment/index.rst:280]: WHZ

**Review by aflaxman — APPROVED** [review]

File: docs/source/intervention_models/wasting_treatment/index.rst (hunk 1 above)
Comment (Ninicorn) [reply docs/source/intervention_models/wasting_treatment/index.rst:158]: Excellent! I also know a pediatrician at the center for global child health in Amsterdam specialized in malnutrition, working mostly in Malawi should we need.

File: docs/source/intervention_models/wasting_treatment/index.rst (hunk 2 above)
Comment (Ninicorn) [reply docs/source/intervention_models/wasting_treatment/index.rst:171]: Ya...in the guidelines its called WFL or WFH (weight for length or weight for height)

Comment (Ninicorn) [reply docs/source/intervention_models/wasting_treatment/index.rst:171]: I can add the distinction in the acronym table

File: docs/source/intervention_models/wasting_treatment/index.rst [hunk 9]
```diff
@@ -62,62 +56,234 @@ in 2012. This is the latest National Guideline for the Management of Acute Malnu
...
+    - weight-for-length z-score (used in GBD)
+  * - WHZ 
+    - weight-for-height z-score (used in GBD)
+  * - 
+    - 
+  * - 
+    - 
+  * - 
+    - 
+
```
Comment (beatrixkh) [inline docs/source/intervention_models/wasting_treatment/index.rst:95]: not important, but this table has some random extra rows

File: docs/source/intervention_models/wasting_treatment/index.rst [hunk 10]
```diff
@@ -62,62 +56,234 @@ in 2012. This is the latest National Guideline for the Management of Acute Malnu
...
+:underline:`Classify SAM with or without medical complications`
 
-Infants 0-6 months: Recent weight loss or failure to gain weight OR Ineffective feeding (attachment, position, and suckling)
+* Any grade of bilateral pitting oedema (+, ++ or +++) OR
+* WHZ < -3 zscore OR
+* Recent weight loss or failure to gain weight OR
+* Ineffective feeding (attachment, positioning and suckling) directly observed for 15-20, minutes, ideally in supervised separate area OR
+* Presence of any of the following medical complications:
 
+  - Poor appetite
```
Comment (beatrixkh) [inline docs/source/intervention_models/wasting_treatment/index.rst:176]: what combination of these is required for a diagnosis? for example, just "dehydration" wouldn't count as SAM, (right?) but all of these combined sounds very scary. 

perhaps the answer to this isnt directly applicable to our model, though

File: docs/source/intervention_models/wasting_treatment/index.rst [hunk 11]
```diff
@@ -145,31 +311,96 @@ Without medical complications
...
 Vivarium Modeling Strategy
 --------------------------
 
+.. image:: treatment_diagram.svg
+   :alt: Compartmental diagram with treatment
+
+
+For **MAM** 0-6 months
+
+ - :math:`r3 =  r3_{ux}\times(1-C_{mam_0-6}) + r3_{tx}\times(C_{mam_0-6})`
```
Comment (beatrixkh) [inline docs/source/intervention_models/wasting_treatment/index.rst:366]: so we will need to look for effect sizes for the different treatments, and then they all effect either **i** or **r**?

**Review by beatrixkh — APPROVED** [review]

File: docs/source/intervention_models/wasting_treatment/index.rst (hunk 10 above)
Comment (Ninicorn) [reply docs/source/intervention_models/wasting_treatment/index.rst:176]: Ya- I think for our purposes...we will just use WHZ and oedema (both gbd has) and we might need to discuss the bias of our model vs. diagnosis in real life...

File: docs/source/intervention_models/wasting_treatment/index.rst (hunk 11 above)
Comment (Ninicorn) [reply docs/source/intervention_models/wasting_treatment/index.rst:366]: YES ! I will need to look at treatment outcomes from programmes for those 5 treatment interventions.

**Pull request merged**